go/parquet/pqarrow/encode_arrow_test.go

```diff
@@ -126,7 +126,7 @@ func makeDateTimeTypesTable(mem memory.Allocator, expected bool, addFieldMeta bo
 }
 
 func TestWriteArrowCols(t *testing.T) {
-	mem := memory.NewCheckedAllocator(memory.DefaultAllocator)
+	mem := memory.NewCheckedAllocator(memory.NewGoAllocator())
```

Comment:
why the switch to `NewGoAllocator` over `DefaultAllocator` ?

Reply:
I just wanted to _make sure_ while whacking ~moles~ leaks, will test and switch to the `DefaultAllocator ` when it's all over :)